app/controllers/constituencies_controller.rb

```diff
@@ -31,7 +31,10 @@ def show
   def current
     letter_data = Parliament::Request.new.constituencies.current.a_z_letters.get
 
-    @constituencies = Parliament::Request.new.constituencies.current.get.sort_by(:name)
+    data = Parliament::Request.new.constituencies.current.get
+
+    @constituencies = data.filter('http://id.ukpds.org/schema/ConstituencyGroup')
+    @constituencies = @constituencies.sort_by(:name)
```

Comment:
I've had a think about this and I'm not sure how I would do this.  When we filter, we loop through the nodes and push them into the correct array depending on their type.  If we wanted to sort at the same time, I think we would still have to loop through each existing array to sort the name node being added to the correct place.  Would this make things quicker?